doc/CATEGORY.md

```diff
@@ -0,0 +1,30 @@
+## Category
+Category has been splits into 3 types: folder, playlist and tag. These will make the management of entity more easier.
```

Comment:
cái này có nghĩa là gì anh  **3 types: folder, playlist and tag**

Reply:
Anh có biết đâu. Trong document uiza đó em. Lôi ra đấy .